src/modules/tools/extruder/Extruder.cpp

```diff
@@ -544,7 +544,7 @@ void Extruder::on_gcode_execute(void *argument)
             this->target_position += this->travel_distance;
             this->en_pin.set(0);
 
-        } else if (gcode->g == 0 || gcode->g == 1) {
+        } else if (gcode->g == 0 || gcode->g == 1 || gcode->g == 2 || gcode->g = 3) {
```

Comment:
Also watch for assignment instead of comparison at the end there! gcode->g
= 3... Compiler should spit a warning about that